README.md

```diff
@@ -90,7 +112,7 @@ Which of the following represents a correct workflow for submitting a PR on a no
 Select 1:
 ```
 [] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
-[] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+[x] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
```

Comment:
This would clone down the WDI Instructors' repo, to which you would not be able to push any changes. You want the first choice, which clones down your fork.